src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -212,7 +211,7 @@ private async Task StartAsyncCore(TransferFormat transferFormat, CancellationTok
 
                 Log.Starting(_logger);
 
-                await SelectAndStartTransport(transferFormat, cancellationToken);
+                await SelectAndStartTransport(_transferFormat, cancellationToken);
```

Comment:
All public API needs `.ForceAsync()`

Also, you've removed the logger scope

Reply:
I'm surprised no tests failed after removing the logger scope. It'd be nice if we tested the `ForceAsync()` stuff too.